src/main/java/com/redhat/cloud/policies/app/rest/PolicyCrudService.java

```diff
@@ -249,7 +249,12 @@ public Response getPoliciesForCustomer() {
         Page<Policy> page;
         try {
             Pager pager = PagingUtils.extractPager(uriInfo);
-            page = Policy.pagePoliciesForCustomer(entityManager, user.getAccount(), pager);
+
+            if (user.getOrgId() != null) {
+                page = Policy.pagePoliciesForCustomer(entityManager, user.getOrgId(), pager);
+            } else {
+                page = Policy.pagePoliciesForCustomer(entityManager, user.getAccount(), pager);
```

Comment:
Most methods from `Policy` need to be duplicated. The old ones should continue dealing with the account while the new ones should deal with the `orgId`. The `account` and the `orgId` are different things (and will coexist for a while). They should never be mixed at any place in the code.

Reply:
The `Policy` class also needs the `orgId` field, it is currently missing.

Reply:
done